src/Relation/Binary/Lattice.agda

```diff
@@ -108,10 +126,10 @@ record BoundedJoinSemilattice c ℓ₁ ℓ₂ : Set (suc (c ⊔ ℓ₁ ⊔ ℓ
 
   open IsBoundedJoinSemilattice isBoundedJoinSemilattice public
 
-  joinSemiLattice : JoinSemilattice c ℓ₁ ℓ₂
-  joinSemiLattice = record { isJoinSemilattice = isJoinSemilattice }
```

Comment:
For this and for `meetSemiLattice` you should create a synonym and mark the original as deprecated, so as to maximise backwards compatability. (see `asymmetric` in `StrictPartialOrder` for an example)

Reply:
I see. Will do it soon. This is mainly for name consistency. Is that a good reason?

Reply:
Yup!